terraform/modules/front_door/variables.tf

```diff
@@ -53,6 +53,12 @@ variable "restrict_by_ip" {
   description = "True if access to cloudfront should be restricted by ip, e.g. before release or for a test environment"
 }
 
+variable "include_test_ips" {
+  type        = bool
+  default     = true

```

Comment:
It's this way round with the default because it made slightly more sense given that post switchover everything that remains restricted will want to include test things.

Reply:
Minor thing - the variable needs to move up in the file in order to keep things in alphabetical order 😀

Reply:
Ah, of course - I put it next to the thing it was linked to while writing. Obviously I should have come up with a name that would keep it there?

Reply:
Have actually thought of such a name because it makes way more sense to me to have these variables next to each other.